roles/network_plugin/calico/rr/tasks/main.yml

```diff
@@ -50,16 +50,15 @@
   command: |-
     {{ bin_dir }}/etcdctl \
```

Comment:
I dont think we have etcdctl available on the RR's so we'll need to copy that over

Reply:
oh!! I'll change it back to delegate

Reply:
fixed